server/index.js

```diff
@@ -11,8 +11,11 @@ app.use('/popular', express.static(path.join(__dirname, '../client/dist')));
 app.use(bodyParser.json());
 
 app.get('/popular/:id', (req, res) => {
-  const { id } = req.params;
-  findRestaurant(id, (err, data) => {
+  const id = req.params;
```

Comment:
i think you can skip line 15 by using:
const id = req.params.id;

Reply:
I still have to convert the id that comes in as ':1' to a number and I did an extra step to get the object value because eslint doesn't like const id = req.params.id